### PR TITLE
feat(content): Spaceport news for various pug planets

### PR DESCRIPTION
**Pull request by tibetiroka**
**Content (Artwork / Missions / Jobs)**

## Summary
Adds various spaceport news to:
 - Deneb, after the pug left
 - Pug-occupied systems after the pug left
 - Planets occupied by the Pug during FW.

Each one gets separate news entries.

Also, I added these in a separate file, hoping that #8902 would get merged with this much extra content in the pug file (or, at least, that this wouldn't just get dumped in there).

## Save File
[Space Shuttle~deneb after fw.txt](https://github.com/endless-sky/endless-sky/files/11755962/Space.Shuttle.deneb.after.fw.txt)
[Space Shuttle~pug during fw.txt](https://github.com/endless-sky/endless-sky/files/11755963/Space.Shuttle.pug.during.fw.txt)

### CODE REVIEW

**Comment by lumbar527** [comment]
On the topic of #8902, I think this should have its own file. The `pug.txt` file is already rather long, and if this gets merged, I think #8902 should also be merged.

File: data/pug/pug news.txt [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+	message
+		word
+			"Can you imagine"
+			"Did you know"
+		word
+			" that "
+		word
+			"this entire"
+			"this whole"
+			"all this"
```
Comment (EjoThims) [inline data/pug/pug news.txt:80]: This one really doesn't make any good sentences with any of the follow up options.

File: data/pug/pug news.txt [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+		government Neutral
+		system Deneb
+	name
+		word
+			"Reporter"
+			"Journalist"
+			"Correspondent"
+	message
+		word
+			"I don't think there is anything "
```
Comment (EjoThims) [inline data/pug/pug news.txt:185]:
```suggestion
			"I don't think there is anything"
```

**Review by EjoThims — CHANGES_REQUESTED** [review]

File: data/pug/pug news.txt (hunk 1 above)
Comment (tibetiroka) [reply data/pug/pug news.txt:80]: I'll try to think up something better.

Comment (tibetiroka) [reply data/pug/pug news.txt:80]:
```suggestion
			"every part of this"
```

Is this better?

Comment (lumbar527) [reply data/pug/pug news.txt:80]: I think so.

File: data/pug/pug news.txt [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+		word
+			"!"
+
+news "conspiracy theorist on post-fw Deneb"
+	location
+		government Neutral
+		system Deneb
+	name
+		word
+			"Conspiracy Theorist"
```
Comment (roadrunner56) [inline data/pug/pug news.txt:122]: Should the second word be capitalized here?

File: data/pug/pug news.txt [hunk 4]
```diff
@@ -0,0 +1,363 @@
...
+			"believe otherwise"
+			"always love when I report from here"
+			"still love to see me here"
+			"like the sound of an alien history"
+			"find it fascinating"
+		word
+			". Anyway, I "
+		word
+			"have to go now"
+			"'m off to the studio"
```
Comment (roadrunner56) [inline data/pug/pug news.txt:216]: This would result in `Anyway, I 'm off to the studio`.

**Review by roadrunner56 — COMMENTED** [review]
Github submitted this as 2 reviews for some reason, sorry.

File: data/pug/pug news.txt (hunk 4 above)
Comment (lumbar527) [inline data/pug/pug news.txt:216]:
```suggestion
			". Anyway, I"
		word
			" have to go now"
			"'m off to the studio"
```
Potential solution to [above review](https://github.com/endless-sky/endless-sky/pull/8911/files#r1233103850).

File: data/pug/pug news.txt (hunk 3 above)
Comment (tibetiroka) [reply data/pug/pug news.txt:122]:
```suggestion
			"Conspiracy theorist"
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

File: data/pug/pug news.txt [hunk 5]
```diff
@@ -0,0 +1,363 @@
...
+
+news "conspiracy theorist on post-fw Deneb"
+	location
+		government Neutral
+		system Deneb
+	name
+		word
+			"Conspiracy theorist"
+			"Critical Thinker"
+			"Truth Seeker"
```
[the remark this answers is not in the document]
Comment (tibetiroka) [reply data/pug/pug news.txt:124]: I think this also works as a sort of self-appointed title.

File: data/pug/pug news.txt [hunk 6]
```diff
@@ -0,0 +1,363 @@
...
+			"always love when I report from here"
+			"still love to see me here"
+			"like the sound of an alien history"
+			"find it fascinating"
+		word
+			". Anyway, I"
+		word
+			" have to go now"
+			"'m off to the studio"
+			"have to get going"
```
[the remark this answers is not in the document]
Comment (tibetiroka) [reply data/pug/pug news.txt:217]: Oh yeah, forgot to fix those.

**Review by roadrunner56 — APPROVED** [review]

File: data/pug/pug news.txt [hunk 7]
```diff
@@ -0,0 +1,363 @@
...
+		word
+			"the people"
+			"my viewers"
+		word
+			" "
+		word
+			"believe otherwise"
+			"always love when I report from here"
+			"still love to see me here"
+			"like the sound of an alien history"
```
Comment (W1zrad) [inline data/pug/pug news.txt:210]: This feels a little weird to say out loud. Thoughts on this instead?
```suggestion
			"want me here, so here I am"
```

**Review by W1zrad — APPROVED** [review]
Looks good!

**Pull request merged**